### DISCUSSION TITLE
how to create paid watchface in the Zepp portal

### DISCUSSION THREAD

**Discussion by syscoZ** [zepp-health/zeppos-forum#34] [Q&A] [answered]
Hi, I have developed some watchfaces and all of them are free, now I want to make more complex ones that are paid, but
I can't find any option in the Zepp development portal for paid watchfaces.

**Comment by sirAvent** [comment, marked as answer]
Here's the Merchant Service page on the Zepp Developer Console.
[https://console.zepp.com/#/account/merchant-certify](https://console.zepp.com/#/account/merchant-certify)

The service is not available overseas yet (only in China). It is being worked on and will come in the future.
